easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb

```diff
@@ -2281,6 +2284,9 @@ exts_list = [
     ('HiddenMarkov', '1.8-11', {
         'checksums': ['4a1614249eee9f428bc182ea9ced443dff4eafa7babf4259c720e5b4da2d08fa'],
     }),
+    ('HiddenMarkov', '1.8-11', {
```

Comment:
@Flamefire Looks like you accidentally copied the entry above here?

Reply:
Thanks. Fixed up the commit.